### ISSUE TITLE
Cannot read property 'mode' of undefined

### ISSUE THREAD

**Issue by alexsalex** [nitaybz/homebridge-tado-ac-old#15]
```
[7/20/2017, 2:36:30 AM] [Tado] The current temperature display unit is ºF
/opt/node/lib/node_modules/homebridge-tado-ac/index.js:175
            accessory.log("Current zone mode is " + obj.setting.mode);
                                                               ^

TypeError: Cannot read property 'mode' of undefined
    at IncomingMessage.<anonymous> (/opt/node/lib/node_modules/homebridge-tado-ac/index.js:175:64)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

When I turn ON in tado app, and then run HomeKit:
```
Jul 20 02:42:39 HomeBridge homebridge[1701]: /opt/node/lib/node_modules/homebridge-tado-ac/index.js:239
Jul 20 02:42:39 HomeBridge homebridge[1701]: if (JSON.stringify(obj.setting.power).match("OFF")) {
Jul 20 02:42:39 HomeBridge homebridge[1701]: ^
Jul 20 02:42:39 HomeBridge homebridge[1701]: TypeError: Cannot read property 'power' of undefined
Jul 20 02:42:39 HomeBridge homebridge[1701]: at IncomingMessage.<anonymous> (/opt/node/lib/node_modules/homebridge-tado-ac/index.js:239:43)
Jul 20 02:42:39 HomeBridge homebridge[1701]: at emitNone (events.js:91:20)
Jul 20 02:42:39 HomeBridge homebridge[1701]: at IncomingMessage.emit (events.js:185:7)
Jul 20 02:42:39 HomeBridge homebridge[1701]: at endReadableNT (_stream_readable.js:974:12)
Jul 20 02:42:39 HomeBridge homebridge[1701]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Jul 20 02:42:39 HomeBridge homebridge[1701]: at process._tickCallback (internal/process/next_tick.js:98:9)
Jul 20 02:42:40 HomeBridge systemd[1]: homebridge.service: main process exited, code=exited, status=1/FAILURE
Jul 20 02:42:40 HomeBridge systemd[1]: Unit homebridge.service entered failed state.
Jul 20 02:42:50 HomeBridge systemd[1]: homebridge.service holdoff time over, scheduling restart.
Jul 20 02:42:50 HomeBridge systemd[1]: Stopping Node.js HomeKit Server...
```

**Comment by jadomski** [comment]
This appears to be the same issue as #14

**Comment by jetmcquack** [comment]
yes for me too, just a different property, it seems it's randomly generating the one and the other.. this time i got this one..
```
TypeError: Cannot read property 'mode' of undefined
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-tado-ac/index.js:169:64)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by nitaybz** [comment]
Duplicate of #14 
fixed in version 1.3.1
